packages/loot-core/src/server/main.ts

```diff
@@ -1776,6 +1776,10 @@ handlers['set-server-url'] = async function ({ url, validate = true }) {
   if (url == null) {
     await asyncStorage.removeItem('user-token');
   } else {
+    if (url.endsWith('/')) {
+      url = url.slice(0, -1);
+    }
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏there could be multiple trailing slashes

```suggestion
    url = url.replace(/\/+$/, '');
```